src/ssl_asn1.c

```diff
@@ -36,6 +36,10 @@
     #endif
 #else
 
+#ifndef UINT8_MAX
+    #define UINT8_MAX 255
```

Comment:
Put a WOLFSSL_MAX_8BIT into types.h.

Reply:
@SparkiDev will do.

would you like for me to also replace all the other `255` magic numbers with `WOLFSSL_MAX_8BIT`?

For example: [ssl.c](https://github.com/wolfSSL/wolfssl/blob/c8f56f035f4058a9fd6c73a8bce9c27ea2ad3dea/src/ssl.c#L2174)

Reply:
Yes

Reply:
@SparkiDev what do you think of [this check](https://github.com/gojimmypi/wolfssl/blob/1ba913543f5deb81c0c566ea0acb6e610ea946b8/src/ssl_asn1.c#L294) causing Jenkins failures? See slack jenkins thread.

Reply:
The tag value is an int as it puts stuff above 8-bits.
Remove check and just take the bottom 8 bits.

Reply:
Hi @SparkiDev thanks for that information. I've added a `ASN_IMPLICIT_TAG_MASK` instead of using the new `WOLFSSL_MAX_8BIT` for the `SetImplicit()` value and expanded the comment regarding the upper bits.

I've only updated the `255` values  to `WOLFSSL_MAX_8BIT` where there's an obvious byte-type value. Other instances of the magic value of 255 should have more intuitive names. (e.g. lengths, etc.)